doc/api/stream.md

```diff
@@ -341,136 +553,140 @@ reader.on('end', () => {
 });
 ```
 
-Note that [`process.stderr`][] and [`process.stdout`][] are never closed until
-the process exits, regardless of the specified options.
+*Note*: The [`process.stderr`][] and [`process.stdout`][] Writable streams are
+never closed until the Node.js process exits, regardless of the specified
+options.
```

Comment:
I think they both leaks.  If there is an error, you need to go and close every stream. That's why we use @mafintosh `pump`